modules/ROOT/pages/import/visual-tour.adoc

```diff
@@ -2,6 +2,9 @@
 :description: This section provides an overview of the Import user interface.
 = Visual tour
 
+Once you have connected to a data source, Importer's interface is the same regardless of your data source.
+It is divided into three parts, the files panel where you provide the files, the data model panel where you draw your model, and the details panel where you map your model to your files.
```

Comment:
I think we could do with renaming this to the data source panel in Docs, its now called this in the product. It will show tables or files depending on source.


in the mapping panel, for simplicity everthing is referred to as a table that you map to rather than distinguishing between file and table.

Reply:
Ouch, this makes it a little awkward since we have a file called "file-provision". I'll try to figure out a workaround to avoid having to rename that and create redirects..

Reply:
I've had a rewrite, reads better now!